BitcoinArmory.spec.in

```diff
@@ -0,0 +1,43 @@
+# disable debug package, otherwise nasty build error in fedora >= 23
```

Comment:
As mentioned elsewhere, I'm not familiar with Fedora's build system. Do the spec files help compile Fedora packages?